crates/turborepo-scm/src/git.rs

```diff
@@ -76,7 +77,8 @@ fn execute_git_command(
     args: &[&str],
     pathspec: &str,
 ) -> Result<Vec<u8>, Error> {
-    let mut command = Command::new("git");
+    let git_binary = which("git")?;
```

Comment:
We're going to use `git` a lot. We should perhaps memoize this and make it easy to use. Can do in a follow-up though.